sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/SignatureAlgorithm.cs

```diff
@@ -35,55 +36,60 @@ public SignatureAlgorithm(string value)
-        /// Gets an RSASSA-PSS using SHA-256 and MGF1 with SHA-256 <see cref="SignatureAlgorithm"/>.
+        /// Gets an RSASSA-PSS using SHA-256 and MGF1 with SHA-256 <see cref="SignatureAlgorithm"/> as described in <see href="https://tools.ietf.org/html/rfc7518"/>.
         /// </summary>
         public static SignatureAlgorithm PS256 { get; } = new SignatureAlgorithm(PS256Value);
 
         /// <summary>
-        /// Gets an RSASSA-PSS using SHA-384 and MGF1 with SHA-384 <see cref="SignatureAlgorithm"/>.
+        /// Gets an RSASSA-PSS using SHA-384 and MGF1 with SHA-384 <see cref="SignatureAlgorithm"/> as described in <see href="https://tools.ietf.org/html/rfc7518"/>.
         /// </summary>
         public static SignatureAlgorithm PS384 { get; } = new SignatureAlgorithm(PS384Value);
 
         /// <summary>
-        /// Gets an RSASSA-PSS using SHA-512 and MGF1 with SHA-512 <see cref="SignatureAlgorithm"/>.
+        /// Gets an RSASSA-PSS using SHA-512 and MGF1 with SHA-512 <see cref="SignatureAlgorithm"/> as described in <see href="https://tools.ietf.org/html/rfc7518"/>.
         /// </summary>
         public static SignatureAlgorithm PS512 { get; } = new SignatureAlgorithm(PS512Value);
 
         /// <summary>
-        /// Gets an ECDSA with a P-256 curve <see cref="SignatureAlgorithm"/>.
+        /// Gets an ECDSA with a P-256 curve <see cref="SignatureAlgorithm"/> as described in <see href="https://tools.ietf.org/html/rfc7518"/>.
         /// </summary>
         public static SignatureAlgorithm ES256 { get; } = new SignatureAlgorithm(ES256Value);
 
         /// <summary>
-        /// Gets an ECDSA with a P-384 curve <see cref="SignatureAlgorithm"/>.
+        /// Gets an ECDSA with a P-384 curve <see cref="SignatureAlgorithm"/> as described in <see href="https://tools.ietf.org/html/rfc7518"/>.
         /// </summary>
         public static SignatureAlgorithm ES384 { get; } = new SignatureAlgorithm(ES384Value);
 
         /// <summary>
-        /// Gets an ECDSA with a P-521 curve <see cref="SignatureAlgorithm"/>.
+        /// Gets an ECDSA with a P-521 curve <see cref="SignatureAlgorithm"/> as described in <see href="https://tools.ietf.org/html/rfc7518"/>.
         /// </summary>
         public static SignatureAlgorithm ES512 { get; } = new SignatureAlgorithm(ES512Value);
 
         /// <summary>
-        /// Gets an ECDSA with a secp256k1 curve <see cref="SignatureAlgorithm"/>.
+        /// Gets an ECDSA with a secp256k1 curve <see cref="SignatureAlgorithm"/> as described in <see href="https://tools.ietf.org/html/rfc7518"/>.
         /// </summary>
         public static SignatureAlgorithm ES256K { get; } = new SignatureAlgorithm(ES256KValue);
 
+        /// <summary>
+        /// Gets an ECDSA with a secp256k1 curve <see cref="SignatureAlgorithm"/> as described in <see href="https://tools.ietf.org/html/rfc8032"/>.
+        /// </summary>
+        public static SignatureAlgorithm EdDSA { get; } = new SignatureAlgorithm(EdDSAValue);
```

Comment:
@KrzysztofCwalina, @tg-msft: I realize this is a departure from our typically casing, but prior you allowed `ToRSA` and `ToECDsa` because it matched the .NET types. There is a (long-standing) ask to add EdDSA (as its cased in the industry) which I imagine would follow a similar naming convention. As such, is this fine as "EdDSA" or should it be "EdDsa"?

For context, it stands for "Edwards-curve Digital Signature Algorithm".

Reply:
I am fine with EdDSA for the same reason: this is how .NET cases it.